Index: shop/templates/shop/login.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{% extends 'shop/index.html' %}\r\n{% load static %}\r\n{% block css %}\r\n    <link rel=\"stylesheet\" href=\"{% static 'css/login.css' %}\">\r\n{% endblock css %}\r\n{% block content %}\r\n    <form class=\"login\">\r\n        {% csrf_token %}\r\n        <div class=\"login-container\">\r\n            <h1>Zaloguj się</h1>\r\n            <hr>\r\n            <label for=\"username\">Login</label>\r\n            <input type=\"text\" placeholder=\"Podaj nazwę użytkownika\" name=\"username\" id=\"username\" required>\r\n\r\n            <label for=\"password\">Hasło</label>\r\n            <input type=\"password\" placeholder=\"Wpisz hasło\" name=\"password\" id=\"password\" required>\r\n            <br /><hr><br />\r\n\r\n            <button type=\"submit\" class=\"login-button\">Zaloguj się</button>\r\n        </div>\r\n    </form>\r\n{% endblock content %}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/shop/templates/shop/login.html b/shop/templates/shop/login.html
--- a/shop/templates/shop/login.html	
+++ b/shop/templates/shop/login.html	
@@ -1,8 +1,6 @@
 {% extends 'shop/index.html' %}
-{% load static %}
-{% block css %}
-    <link rel="stylesheet" href="{% static 'css/login.css' %}">
-{% endblock css %}
+{% load static %} {% block css %}
+    <link rel="stylesheet" href="{% static 'css/login.css' %}"> {% endblock css %}
 {% block content %}
     <form class="login">
         {% csrf_token %}
